src/guide/components/events.md

```diff
@@ -444,8 +444,8 @@ Each v-model will sync to a different prop, without the need for extra options i
 
 ```vue-html
 <UserName
-  v-model:first-name="firstName"
-  v-model:last-name="lastName"
+  v-model:first-name="name1"
```

Comment:
`name1` and `name2` feels a bit too contrived, how about just `first` and `last`?

Reply:
Agreed. Never really liked `name1` and `name2` but I was having a bit of a mental block trying to think of something better.